### DISCUSSION TITLE
Aws s3 bucket.

### DISCUSSION THREAD

**Discussion by mnsyawal** [awsugid/forum#36] [Q&A]
Izin bertanya, adakah yang pernah buat website penyimpanan, yang datanya tersimpan di s3 bucket

**Comment by saiqulhaq** [comment]
maksudnya mau bikin file hosting dan storage nya pake S3?

**Reply by mnsyawal** [reply]
Iaa kak

**Reply by saiqulhaq** [reply]
harusnya udah banyak resource di internet klo pemakaian S3 untuk static websites
kamu pake framework atau bikin sendiri?
tinggal di buat public access jg bisa
